src/Ucommerce.Avenue.Umbraco/Api/AvenueClothingApiSearchController.cs

```diff
@@ -27,7 +27,7 @@ public IHttpActionResult Search([FromBody] SearchRequest request)
                             && (
                                 p.Sku.Contains(request.Keyword)
                                 || p.Name.Contains(request.Keyword)
-                                || p.DisplayName.Contains(request.Keyword)
+                                || p.DisplayName == Match.FullText(request.Keyword)
                                 || p.LongDescription.Contains(request.Keyword)
                                 || p.ShortDescription.Contains(request.Keyword)
```

Comment:
Long and Short description are also full test indexed: https://github.com/Ucommercenet/Ucommerce/blob/16713971f2d15a48e108a54a7decaa4fbd937365/src/Ucommerce.Search/Definitions/DefaultProductsIndexDefinition.cs#L33